stories/topics/proc-aap-gcp-backup-and-restore.adoc

```diff
@@ -0,0 +1,123 @@
+[id="proc-aap-gcp-backup-and-restore"]
+
+= Ansible Automation Platform Backup and Restore Process
+
+In order to backup and restore your AAP deployment, it is vital to have your existing AAP Admin Secret name and value recorded somewhere safe. In addition to this, it is also extremely important that regular manual backups of the Cloud SQL DB instance and filestore backups are taken to ensure a deployment can be restored as close as possible to its previous working state. The playbooks backup and restore provides backup and restore support for the AAP on GCP foundation deployment. 
+
+**Please not that the restore process will deploy a new deployment with the filestore and SQL DB instance being restored to the backup specified.**
+
+Follow the steps below to ensure a smooth backup and recovery process.
```

Comment:
The backup process involves taking a backup of the Cloud SQL DB and filestore instances at a given point in time.
The backup playbook require an active {AAPonGCP} foundation deployment to be running.
 
The following instructions describe how to run the backup playbook.